src/mono/wasm/wasm.proj

```diff
@@ -35,32 +50,42 @@
                            OutputPath="$(WasmPInvokeTablePath)" />
   </Target>
 
-  <Import Project="timezones.props" Condition="'$(TargetOS)' == 'Browser'" />
-  <UsingTask TaskName="CompileTimeZoneData" 
-    AssemblyFile="$(GenerateWasmBundlesAssemblyPath)"/>
-  <Target Name="LoadTimeZones" DependsOnTargets="CheckEnv">
+  <UsingTask TaskName="CompileTimeZoneData"
+    AssemblyFile="$(WasmBuildTasksAssemblyPath)"/>
+  <Target Name="CompileTimeZones">
+    <ItemGroup>
+      <TimeZoneToCompile Include="africa" />
+      <TimeZoneToCompile Include="antarctica" />
+      <TimeZoneToCompile Include="asia" />
+      <TimeZoneToCompile Include="australasia" />
+      <TimeZoneToCompile Include="etcetera" />
+      <TimeZoneToCompile Include="europe" />
+      <TimeZoneToCompile Include="northamerica" />
+      <TimeZoneToCompile Include="southamerica" />
+    </ItemGroup>
     <CompileTimeZoneData
-      TimeZones="@(TimeZoneToBundle)" 
-      InputDirectory="$([MSBuild]::NormalizePath('$(GenerateWasmBundleDir)', '..', 'data'))"
-      OutputDirectory="$(ArtifactsObjDir)wasm/data/output" />
+      TimeZones="@(TimeZoneToCompile)"
+      InputDirectory="$([MSBuild]::NormalizePath('$(PkgSystem_Runtime_TimeZoneData)', 'contentFiles', 'any', 'any', 'data'))"
+      OutputDirectory="$(ArtifactsObjDir)wasm/timezones" />
   </Target>
 
   <UsingTask TaskName="GenerateWasmBundle"
-    AssemblyFile="$(GenerateWasmBundlesAssemblyPath)"/>
-  <Target Name="BundleWasm" DependsOnTargets="CheckEnv;LoadTimeZones">
+    AssemblyFile="$(WasmBuildTasksAssemblyPath)"/>
+  <Target Name="BundleTimeZones" DependsOnTargets="CompileTimeZones">
     <PropertyGroup>
       <TimeZonesDataPath>$(NativeBinDir)dotnet.timezones.blat</TimeZonesDataPath>
     </PropertyGroup>
     <GenerateWasmBundle
-      InputDirectory="$(ArtifactsObjDir)wasm/data/output"
+      InputDirectory="$(ArtifactsObjDir)wasm/timezones"
```

Comment:
sad panda when you use forward slashes in msbuild 😭

Reply:
it's the best slash 😸